Library/Homebrew/unpack_strategy.rb

```diff
@@ -6,46 +6,27 @@
 # Module containing all available strategies for unpacking archives.
 module UnpackStrategy
   extend T::Helpers
+  include SystemCommand::Mixin
+  abstract!
+
+  # rubocop:disable Style/MutableConstant
+  UnpackStrategyImpl = T.type_alias { T.all(T::Class[UnpackStrategy], UnpackStrategy::ClassMethods) }
```

Comment:
```suggestion
  UnpackStrategyType = T.type_alias { T.all(T::Class[UnpackStrategy], UnpackStrategy::ClassMethods) }
```

or something? Not a huge fan of `Impl` here.